.travis.yml

```diff
@@ -28,7 +28,7 @@ env:
     # SGX SSL
     - OPENSSL_VERSION=1.1.0j
     - SGXSSL_VERSION=v2.4.1
-    - SGXSSL=/opt/intel/sgxssl
+    - SGX_SSL=/opt/intel/sgxssl
```

Comment:
I like `SGX_SSL` over `SGXSSL` but it introduces a small inconsistency with other *SSL vars here in this travis file.

Reply:
This was actually misnamed in travis, because other parts of the project use `SGX_SSL`.
And this was the cause that made travis fail, because the pdo crypto lib requires it to compile properly.